test/derived.test.ts

```diff
@@ -231,15 +242,18 @@ test("calculated with addModeDefaults", () => {
 
   // we now change a, which should modify the derived value
   sub0.field("a").setRaw("3");
+  //  expect(changeCount).toBe(1);
```

Comment:
commented expect, should be removed?

Reply:
I was busy debugging this, I fixed it just now.